core/tests/webdriverio_utils/users.js

```diff
@@ -82,16 +82,17 @@ var login = async function(email, useManualNavigation = true) {
   });
 };
 
-var logout = async function() {
+var logout = async function(id = '') {
```

Comment:
I think we need to make this a lot clearer -- it's not clear what "id" means and what is a valid ID. Also this is a pretty specific fix for the splash page only and it doesn't really help us if the same issue appears in other places.

Have you looked at other ways to get a better stack trace for promises? A couple of quick searches point to https://stackoverflow.com/a/57070547 (what is our node version) and https://pavelevstigneev.medium.com/capture-javascript-async-stack-traces-870d1b9f6d39 -- I would suggest making a debugging doc that lists possible ways to get better stack traces and then doing some experimenting (e.g. try and throw an error on purpose somewhere deep down and see if you can get the full stack trace leading to it). That would be better than the current solution, which is rather hacky and special-case.

Reply:
@seanlip I remove id and add full stack trace now.